packages/2018-transportation-systems/src/components/DeclineInRidership/index.js

```diff
@@ -1,25 +1,93 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { css } from 'emotion';
 
-import { CivicStoryCard, Placeholder } from '@hackoregon/component-library';
+import { CivicStoryCard, LineChart } from '@hackoregon/component-library';
+
+import { fetchRidershipOverTime } from '../../state/decline-in-ridership/actions';
+import {
+  isRidershipOverTimePending,
+  getRidershipOverTimeData,
+} from '../../state/decline-in-ridership/selectors';
+
+const sampleSimpleData = [
+  { x: 0, y: 20 },
+  { x: 10, y: 30 },
+  { x: 20, y: 50 },
+  { x: 30, y: 40 },
+];
+
+const cardLoading = css`
+  width: 100%;
+  padding: 50px;
+  text-align: center;
+  background: #EEE;
+`;
+
+const cardError = css`
+  width: 100%;
+  padding: 50px;
+  text-align: center;
+  background: #FDD;
+  border: 1px solid #C99;
+`;
```

Comment:
Not gonna make you do it as part of this PR, but these common loading and error states should both be moved into the component library and redone to look nice.

Reply:
Agreed. Created issue #249.